src/Uno.UI.Runtime.WebAssembly.Compatibility/Uno.UI.Runtime.WebAssembly.Compatibility.csproj

```diff
@@ -36,14 +36,13 @@
 		</PropertyGroup>
 		<ItemGroup>
 			<_OutputFiles Include="@(TargetPathWithTargetPlatformMoniker)" />
-			<_OutputFilesPDB Include="@(TargetPathWithTargetPlatformMoniker-&gt;'%(RecursiveDir)%(Filename).pdb)" />
+			<_OutputFilesPDB Include="@(TargetPathWithTargetPlatformMoniker-&gt;'%(RecursiveDir)%(Filename).pdb')" />
```

Comment:
Given this is only used by the copy task, it should be safe to remove as well.

Reply:
Oh yeah that's true, thanks @Xiaoy312 !